### PR TITLE
Content: added extra front-matter options to example page

### PR DESCRIPTION
**Pull request by anne-usmedia**
- Added possible front-matter options to example page (/examples/basic/front-matter/)
- Fixed front-matter typo in head.html

For https://linear.app/usmedia/issue/CUE-340

### CODE REVIEW

File: hugo/layouts/partials/site/head.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 {{ else }}{{ with .Site.Params.description -}}{{- . | $.RenderString | plainify -}}
 {{ end }}{{ end }}{{ end -}}">
 {{ hugo.Generator }}
-{{ if and (eq (getenv "HUGO_ENV") "production") (ne .Params.noindex true) (ne .Type "examples") }}
+{{ if and (eq (getenv "HUGO_ENV") "production") (ne .Params.no_index true) (ne .Type "examples") }}
```
Comment (jpluscplusm) [inline hugo/layouts/partials/site/head.html:8]: In the current tip of alpha:

```
$ git grep no_index | wc -l
3
$ git grep noindex | wc -l
16
```

I suggest either splitting this PR into 2 (moving the noindex fix into its own PR, with fixups for all the current in-repo uses of `noindex`), or including all those fixups in this PR.

Either way, merging this PR as it stands right now, I /think/ we'll have a bunch of pages (e.g. example blog posts) getting indexed where they previously weren't.

**Review by jpluscplusm — CHANGES_REQUESTED** [review]

File: hugo/layouts/partials/site/head.html (hunk 1 above)
Comment (anne-usmedia) [reply hugo/layouts/partials/site/head.html:8]: Ah yes, I missed those, sorry! Both the no_index and noindex are being used apparently. Will let you know when it's fixed!

**Review by jpluscplusm — APPROVED** [review]
LGTM, assuming zero `noindex` instances remain after this PR :-)
Importing now.

**Comment by jpluscplusm** [comment]
Imported as https://review.gerrithub.io/c/cue-lang/cuelang.org/+/1172301

**Comment by jpluscplusm** [comment]
Closed via 46dbe39c6e03bedc3e28c3a254b0d86044974071
